docs/tutorials/troubleshooting/deploy-vm.md

```diff
@@ -0,0 +1,54 @@
+# Deploy VM
+This page is dedicated to diagnose issues that can arise during the deployment of virtual machines (VMs). Refer to [Deploy vm](https://vm-operator.readthedocs.io/en/stable/tutorials/deploy-vm/) on how to deploy a virtual machine.
+
+## Troubleshoot step
+### Describe Virtual Machine k8s resource
+After you ssh into the supervisor cluster control plane VM, run `kubectl describe vm <vm-name> -n <namespace-name>` command. Check `VM.Status.Conditions` as seen in sample output below.
+
+```console
+$ kubectl describe vm <vm-name> -n <namespace-name>
+
+...
+Status:
+  Bios UUID:              4239fba9-7390-1130-478a-e559a25f91e6
+  Change Block Tracking:  false
+  Conditions:
+    Last Transition Time:  2022-07-06T00:43:47Z
+    Status:                Unknown
+    Type:                  GuestCustomization
+    Last Transition Time:  2022-07-06T00:42:41Z
+    Status:                True
+    Type:                  VirtualMachinePrereqReady
+    Last Transition Time:  2022-07-06T00:43:47Z
+    Message:               VMware Tools is not running
+    Reason:                VirtualMachineToolsNotRunning
+    Severity:              Error
+    Status:                False
+    Type:                  VirtualMachineTools
+...
+```
+
+## Error and fix
+This section focuses on leveraging the information provided by the `VM.Status.Conditions` field to diagnose the root cause of errors and provide solutions. By analyzing which conditions showed `Status` as `False`/`Unknown` and `Reason`, you can troubleshoot issues depending on below scenarios.
+
+### VirtualMachinePrereqReady
+- Reason `VirtualMachineClassNotFound`/`VirtualMachineClassBindingNotFound`: this means that the VM class has not been associated to the Supervisor namespace.
+    - **Fix**: Attach VM Class to your namespace. Refer to [VMClass](https://vm-operator.readthedocs.io/en/stable/concepts/workloads/vm-class/) for details.
+- Reason `VirtualMachineImageNotFound`: this means that the VirtualMachine Image doesn't exist in the supervisor cluster, which means that this VM image doesn't exist in the vCenter, or the content library which it belongs to has not been associated to any of the supervisor namespaces. 
+    - **Fix**: Upload mentioned VM Image to content library and associate content library to your namespace. Refer to [VM Image](https://vm-operator.readthedocs.io/en/stable/concepts/images/) for details.
+- Reason `ContentSourceBindingNotFound`: this means that the content library which contains the specified VM Image has not been associated to the Supervisor namespace. 
```

Comment:
Avoid having deprecated information.

Reply:
But for users still using pre `WCP_VM_Image_Registry` enabled, this is a solid error reason and still lives in our code. https://github.com/vmware-tanzu/vm-operator/blob/main/pkg/vmprovider/providers/vsphere/vmprovider_vm_utils.go#L177

Reply:
I think we wanna keep our documentation forward facing. If we do end up putting this here, maybe call out that this is `Deprecated` ?.